morphir/src/org/finos/morphir/runtime/sdk/BasicsSDK.scala

```diff
@@ -191,4 +191,58 @@ object BasicsSDK {
         ctx.evaluator.handleApplyResult(Type.UType.Unit(()), f1, res1)
       }
   }
+
+  val e: SDKValue = SDKValue.SDKNativeValue(Primitive.Float(scala.math.E))
+
+  val pi: SDKValue = SDKValue.SDKNativeValue(Primitive.Float(scala.math.Pi))
```

Comment:
Several of these are redundant with older implementations from https://github.com/finos/morphir-scala/blob/main/morphir/runtime/src/org/finos/morphir/runtime/quick/Native.scala - please remove those where they overlap

Reply:
Removed the redundant code.